archaius2-core/src/main/java/com/netflix/archaius/annotations/PropertyInfo.java

```diff
@@ -23,6 +23,6 @@
 @Documented
 @Retention(java.lang.annotation.RetentionPolicy.RUNTIME)
 @Target({ElementType.FIELD, ElementType.METHOD})
-public @interface Value {
+public @interface PropertyInfo {
```

Comment:
@ Property is not yet taken. If you want a more uber annotation that allows definition of alternative names and optionally other attributes, how about that?